content/sql/concepts/window-functions/terms/lag/lag.md

```diff
@@ -59,3 +59,41 @@ Output
 | jenna      | black     | 35  | 60           |
 | chris      | smith     | 30  | 35           |
 | dave       | james     | 19  | 30           |
+
+## Example 
+### Using partition by
```

Comment:
The header for the second example, "Example Using partition by," uses inconsistent casing and lacks proper capitalization for `PARTITION BY`.

```suggestion
### Using `PARTITION BY` Clause
```